purchase_requisition_line_description/models/purchase_requisition_line.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+from odoo.fields import first
+
+
+class PurchaseRequisitionLine(models.Model):
+    _inherit = "purchase.requisition.line"
+
+    name = fields.Text(string="Product Description")
+
+    @api.onchange("product_id")
```

Comment:
I'm wondering if we shouldn't change this to a compute function depending on the product_id :thinking:

Reply:
Changes are done.

Reply:
@grindtildeath would appreciate your review.

Reply:
@bizzappdev please have a look at the codecov test results and try to improve the test coverage.

Reply:
@ivs-cetmix Test cases codecov are now proper. There was a line.product_id condition, which was not possible as product_id is a required field, so we have adjusted the compute method. Thanks for pointing it out.

Reply:
@bizzappdev perfect, thank you 👍